src/components/views/SSITabView/index.tsx

```diff
@@ -7,17 +8,18 @@ import {ITabRoute, ITabViewRoute} from '../../../types';
 import SSITabViewHeader from './SSITabViewHeader';
 
 export interface IProps {
+  style?: ViewStyle;
   routes: Array<ITabViewRoute>;
 }
 
 const SSITabView: FC<IProps> = (props: IProps): JSX.Element => {
   const [index, setIndex] = React.useState(0);
   const [routes] = React.useState(props.routes);
-
+  const style = props.style;
   const sceneMap = props.routes.reduce((a: Record<string, any>, v: ITabViewRoute) => ({...a, [v.key]: v.content}), {});
 
   return (
-    <Container>
+    <Container style={{...(style && style)}}>
```

Comment:
is this needed? the tabview just hold some routes, these routes should determine which color of background they need

Reply:
removed this styling